src/ms-review.tex

```diff
@@ -0,0 +1,1786 @@
+the \code{TSMapEstimator} objects with a given input \code{MapDataset}.
+In addition to the model it allows to specify the energy
+bins of the resulting flux and TS maps.
+
+
+\begin{figure}
+	\import{code-examples/generated/}{gp_estimators}
+	\caption{Using the \code{TSMapEstimator} object from \code{gammapy.estimators} to compute a
+        a flux, flux upper limits and TS map. The additional parameters \code{n\_sigma}
+        and \code{n\_sigma\_ul} define the confidence levels (in multiples of the normal distribution width)
+        of the flux error and and flux upper limit maps respectively.
+    }
+    \label{fig*:minted:gp_estimators}
+\end{figure}
+
+\subsection{gammapy.analysis}
+\label{ssec:gammapy-analysis}
+The \code{gammapy.analysis} sub-module provides a  high-level interface for the most
+common use cases identified in \gammaray analyses. The classes and methods
+included can be used in Python scripts, notebooks or as commands within IPython
+sessions. The high-level user interface can also be used to automatise
+workflows driven by parameters declared in a configuration file in YAML format.
+This way a full analysis can be executed via a single command line taking the
+configuration file as input.
+
+The \code{Analysis} class has the responsibility of orchestrating of the workflow
+defined in the configuration \code{AnalysisConfig} objects and triggering the execution of
+the \code{AnalysisStep} classes that defines the common use cases identified. These
+steps include the following: observations selection from the DataStore,  data
+reduction, excess map computation, model fitting, flux-points estimation, and
+light-curves production. The structure of the \code{Analysis} class allows to define
+custom AnalysisStep, so ones could design and configure arbitrarily complex
+wokflows. \todo{the latter is not true for v1.0..., include example YAML file?}
+
+
+\subsection{gammapy.stats}
+\label{ssec:gammapy-stats}
+The \code{gammapy.stats} subpackage contains the fit statistics and associated
+statistical estimators that are commonly used in \gammaray astronomy. In
+general, \gammaray observations count Poisson-distributed events at various sky
+positions, and contain both signal and background events. Estimation of the
+number of signal events is done through likelihood maximization. In \gammapy,
+the fit statistics are Poisson log-likelihood functions normalized like
+chi-squares, i.e., they follow the expression $2 \time\cdot \log{\mathcal{L}}$,
+where $\mathcal{L}$ is
+the likelihood function used. The statistic function used when the expected
+number of background events is known is the \emph{Cash} statistic
+~\citep{Cash}. It is used by datasets using background templates such as the
+\code{MapDataset}. When the number of background events is unknown and an off
+measurement where only background events are expected is used, the statistic
+function is WStat. It is a profile log-likelihood statistic where background
```

Comment:
no reference for WStat (as for the Cash)?

Reply:
As far as I know there is no official reference for `wstat`, we have always linked to some spec page for this....maybe @registerrier knows better?